rpmlint/lint.py

```diff
@@ -81,6 +80,36 @@ def _load_installed_rpms(self, packages):
                 print_warning(f'(none): E: there is no installed rpm "{name}".')
         return existing_packages
 
+    def _load_rpmlintrc(self):
+        """
+        Load rpmlintrc from argument or load up from folder
+        """
+        if self.options['rpmlintrc']:
+            self.config.load_rpmlintrc(self.options['rpmlintrc'])
+        else:
+            # load only from the same folder specname.rpmlintrc or specname-rpmlintrc
+            # do this only in a case where there is one folder parameter or one file
+            # to avoid multiple folders handling
+            rpmlintrc = []
+            if not len(self.options['rpmfile']) == 1:
+                return
+            pkg = self.options['rpmfile'][0]
+            if pkg.is_dir():
+                files = pkg.iterdir()
+            if pkg.is_file():
+                files = pkg.parent.iterdir()
+            for item in files:
```

Comment:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob i think will be better

Reply:
done